packages/filebrowser-extension/src/index.ts

```diff
@@ -848,3 +827,59 @@ function createLauncher(
       return launcher;
     });
 }
+
+/**
+ * Get browser object given file path.
+ */
+function getBrowserForPath(
+  path: string,
+  app: JupyterLab,
+  browser: FileBrowser,
+  tracker: IInstanceTracker<FileBrowser>
+): FileBrowser {
+  const driveName = app.serviceManager.contents.driveName(path);
```

Comment:
We can get the contents manager off of the `browser.manager.services.contents` and remove the `app` dependency. We can also get the default `browser` off of the `IFileBrowserTracker` and remove that dependency.

Reply:
It looks like there is no `manager` property on [`FileBrowser`](https://github.com/jupyterlab/jupyterlab/blob/master/packages/filebrowser/src/browser.ts#L67)?

Reply:
> We can also get the default browser off of the IFileBrowserTracker and remove that dependency.

What would that look like?

Reply:
Sorry, I meant `browser.model.services.contents`.

The tracker has a `defaultBrowser` property: https://github.com/jupyterlab/jupyterlab/blob/master/packages/filebrowser-extension/src/index.ts#L228

So we can remove `browser` from the args of the function and instead get `tracker.defaultBrowser`, I think.